scss/standalone/patterns_navigation-combined.scss

```diff
@@ -0,0 +1,3 @@
+@import 'patterns_navigation';
```

Comment:
Not needed with new approach to spacing?

Reply:
@bartaz removed